README.md

```diff
@@ -54,248 +99,292 @@ $(SRCROOT)/Carthage/Build/iOS/CryptoSwift.framework
+#### vi). Add `NSFaceIDUsageDescription` description in `info.plist`
+
+The iOS Wallet SDK can use FaceID in lieu of fingerprint if the hardware supports it. To support faceID, please include  [NSFaceIDUsageDescription](https://developer.apple.com/documentation/bundleresources/information_property_list/nsfaceidusagedescription) key in your application's `info.plist` file and describe its usage.
+
+**Note: [NSFaceIDUsageDescription](https://developer.apple.com/documentation/bundleresources/information_property_list/nsfaceidusagedescription) key is supported in iOS 11 and later.**
+
 
 ## OST Wallet SDK APIs
-To use Ost wallet sdk use `import OstWalletSdk`
 
-### Initialize the SDK
+### Types of Methods
 
-The SDK can be initialized by calling the `initialize()` API which
-initializes all the required instances and runs migrations of local databases.
+1. `Workflows`: Workflows are the core functions provided by wallet SDK to do wallet related actions. Workflows can be called directly by importing the SDK.
 
-```Swift
-func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
-    do {
-        try OstWalletSdk.initialize(apiEndPoint: <OST_PLATFORM_API_ENDPOINT>)
-     } catch let ostError {
+	* Application must confirm to `OstWorkflowDelegate` protocol. The `OstWorkflowDelegate` protocol defines methods that allow application to interact with `OstWalletSdk`.
 
-     }
-     return true
-}
+
+2. `Getters`: These functions are synchronous and will return the value when requested.
+
+
+
+## Workflows
+
+### 1. initialize
+
+You must initialize the SDK before start using it.
+
+Recommended location to call **OstWalletSdk.initialize()** is in [application](https://developer.apple.com/documentation/uikit/uiapplicationdelegate/1622921-application) method of [UIApplicationDelegate](https://developer.apple.com/documentation/uikit/uiapplicationdelegate). 
+
+```
+OstWalletSdk.initialize(apiEndPoint: String)
 ```
 
-### Set up the device
+| Parameter | Description |
+|---|---|
+| **apiEndPoint** <br> **String**	| OST PLATFORM API ENDPOINT: <br> 1. Sandbox Environment: `https://api.ost.com/testnet/v2/` <br> 2. Production Environment: `https://api.ost.com/mainnet/v2/` |
 
-The `setupDevice` API should be called after user login or signup is successful.
 
-Once the user is logged in, then `setupDevice` should be called every time the app launches, this ensures that the current device is registered before communicating with OST Platform server.<br/><br/>
-**Parameters**<br/>
-&nbsp;_userId: OST Platform user id provided by application server_<br/>
-&nbsp;_tokenId: Token id provided by application server_<br/>
-&nbsp;_delegate: Callback implementation object for application communication_<br/>
 
-```Swift
+<br>
+### 2. setupDevice
```

Comment:
The heading is not rendering as expected.
See here:
https://github.com/ostdotcom/ost-wallet-sdk-ios/blob/readme-update/README.md